x-pack/plugins/synthetics/public/legacy_uptime/state/effects/fetch_effect.test.ts

```diff
@@ -6,12 +6,25 @@
  */
 
 import { call, put } from 'redux-saga/effects';
+import type { IHttpFetchError } from '@kbn/core-http-browser';
 import { fetchEffectFactory } from './fetch_effect';
 import { indexStatusAction } from '../actions';
-import { HttpFetchError } from '@kbn/core/public';
-import { StatesIndexStatus } from '../../../../common/runtime_types';
+import type { StatesIndexStatus } from '../../../../common/runtime_types';
 import { fetchIndexStatus } from '../api';
 
+class HttpFetchError extends Error implements IHttpFetchError {
+  name: string;
+  request: Request;
+  req: Request;
+
+  constructor(message: string, name = 'error', req = {} as any) {
```

Comment:
nit to avoid using `any`
```suggestion
  constructor(message: string, name = 'error', req = {} as unknown as Request) {
```

Reply:
I found a better solution, I've created a method to instantiate IHttpFetchErrors that plugins can use for their tests. It will be exposed in `@kbn/core-http-browser-mocks`.